apps/builder/app/routes/logout.tsx

```diff
@@ -1,11 +1,49 @@
-import type { ActionFunctionArgs } from "@remix-run/server-runtime";
+import { json, type LoaderFunctionArgs } from "@remix-run/server-runtime";
+import createDebug from "debug";
 import { authenticator } from "~/services/auth.server";
-import { loginPath } from "~/shared/router-utils";
+import { builderAuthenticator } from "~/services/builder-auth.server";
+import {
+  getAuthorizationServerOrigin,
+  isBuilderUrl,
+} from "~/shared/router-utils/origins";
 
-export default function Logout() {
-  return null;
-}
+const debug = createDebug("Auth").extend("Logout");
```

Comment:
I haven't tried but I think here you won't get "ws:logout:auth", you will get just "logout:auth"

I think you need double extend here: createDebug("ws").extend('auth').extend("Logout")

and I also think we could create one debug function:
const debug = createDebug("ws")

and then reuse it everywhere, so that there is always "ws" namespace defined